enterprise/tailnet/pgcoord_test.go

```diff
@@ -683,7 +592,13 @@ func TestPGCoordinator_Unhealthy(t *testing.T) {
 	logger := slogtest.Make(t, &slogtest.Options{IgnoreErrors: true}).Leveled(slog.LevelDebug)
 
 	calls := make(chan struct{})
+	// first call succeeds, so that our Agent will successfully connect.
```

Comment:
previously, we started a pipe connection and used the v1 API over it.  This eventually calls `Coordinate()`, but starting the pipe didn't block on `Coordinate()` returning.

With the v2 API, we synchronously call `Coordinate()` from the main test goroutine, so we need to get an initially successful heartbeat to unblock the `Coordinate()` call or this test would deadlock.